addons/apecloud-mysql/scripts/etcd-clean.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-endpoints=${ETCD_SERVER:-'127.0.0.1:2379'}
+if [ -n "$SERVICE_ETCD_ENDPOINT" ]; then
+  endpoints=$SERVICE_ETCD_ENDPOINT
+else
+# local etcd no need to clean
+  exit 0 
+fi
```

Comment:
doesn't set endpoints properly

Reply:
I don't understand what the problem is. etcd-clean.sh only needs to be executed when using an external etcd. This can be determined through SERVICE_ETCD_ENDPOINT. If it is empty, it means that an external etcd is not being used, and the process can be terminated immediately.

Reply:
I see, you're correct.